metals/src/main/scala/scala/meta/internal/worksheets/DecorationWorksheetPublisher.scala

```diff
@@ -40,7 +43,10 @@ class DecorationWorksheetPublisher() extends WorksheetPublisher {
       .map { s =>
         new DecorationOptions(
           s.position().toLsp,
-          new MarkedString("scala", s.details()),
+          new MarkupContent(
+            MarkupKind.MARKDOWN,
+            List(scalaMarkdownTics, s.details, endTics).mkString("\n")
```

Comment:
I was going to do a multiline string here, but scalafmt kept breaking the string causing compilation to fail. I'll report an issue to scalafmt about it.